dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateConstants.java

```diff
@@ -0,0 +1,8 @@
+package com.dotmarketing.portlets.templates.business;
+
+public class TemplateConstants {
+
+    public static String TEMPLATE_DELETE_ERROR = "message.template.full_delete.error";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "TEMPLATE_DELETE_ERROR" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static TEMPLATE_DELETE_ERROR" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make TEMPLATE_DELETE_ERROR a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)